instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerAttributesExtractor.java

```diff
@@ -90,18 +98,20 @@ public static <REQUEST, RESPONSE> HttpServerAttributesExtractorBuilder<REQUEST,
   public void onStart(AttributesBuilder attributes, Context parentContext, REQUEST request) {
     super.onStart(attributes, parentContext, request);
 
-    String forwardedProto = forwardedProto(request);
-    String value = forwardedProto != null ? forwardedProto : getter.getScheme(request);
-    internalSet(attributes, SemanticAttributes.HTTP_SCHEME, value);
-    internalSet(attributes, SemanticAttributes.HTTP_TARGET, getTarget(request));
+    internalUrlExtractor.onStart(attributes, request);
+    internalNetExtractor.onStart(attributes, request);
+
+    if (SemconvStability.emitOldHttpSemconv()) {
+      String forwardedProto = forwardedProto(request);
+      String value = forwardedProto != null ? forwardedProto : getter.getUrlScheme(request);
```

Comment:
@trask the new spec does not mention the forwarded proto at all - without implementing https://github.com/open-telemetry/opentelemetry-specification/issues/3442 in some form we might actually end up emitting conflicting data when `http/dup` is used; see the `HttpServerAttributesExtractorBothSemconvTest` for an example.

Reply:
what do you think about introducing a new opt-in setting that enables overriding both `http.scheme` and `url.scheme` with forwardedProto? (and then deprecating that setting after https://github.com/open-telemetry/opentelemetry-specification/issues/3442)

Reply:
Good point! Will do that